<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenreWidget</class>
 <widget class="QWidget" name="GenreWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>702</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="MySplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>1</number>
     </property>
     <widget class="QTableWidget" name="genres">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Genre</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>21</number>
       </property>
       <item>
        <widget class="QLabel" name="genreName">
         <property name="font">
          <font>
           <family>Helvetica Neue</family>
           <pointsize>22</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="indent">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="infoGroupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTableWidget" name="movies">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="showGrid">
               <bool>false</bool>
              </property>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Movie</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="buttonAddMovie">
                <property name="toolTip">
                 <string>Add Movie</string>
                </property>
                <property name="text">
                 <string>Add Movie</string>
                </property>
                <property name="dropShadow" stdset="0">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="buttonRemoveMovie">
                <property name="toolTip">
                 <string>Remove Current Movie</string>
                </property>
                <property name="text">
                 <string>Remove Movie</string>
                </property>
                <property name="dropShadow" stdset="0">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Double click a genre to rename it, right click to delete. If you want to merge two genres just give them the same name.</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Please keep in mind that the changes you make here (renaming or deleting genres) will be made for every movie.</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MySplitter</class>
   <extends>QSplitter</extends>
   <header>ui/small_widgets/MySplitter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonAddMovie</sender>
   <signal>clicked()</signal>
   <receiver>GenreWidget</receiver>
   <slot>addMovie()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>555</x>
     <y>651</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonRemoveMovie</sender>
   <signal>clicked()</signal>
   <receiver>GenreWidget</receiver>
   <slot>removeMovie()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>651</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addMovie()</slot>
  <slot>removeMovie()</slot>
 </slots>
</ui>
